test.js

```diff
@@ -0,0 +1,4 @@
+ReactDOM.render(
+  <h1>Hello, world!</h1>,
+  document.getElementById('root')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ShonaMac/a11y-testing/pullRequest?prid=8011508)
```suggestion
  document.getElementById("root")
```